docs/usage.md

```diff
@@ -303,7 +303,7 @@ Complexity filtering is primarily a run-time optimisation step. It is not necess
 
 There are currently three options for short-read complexity filtering: [`bbduk`](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/), [`prinseq++`](https://github.com/Adrian-Cantu/PRINSEQ-plus-plus), and [`fastp`](https://github.com/OpenGene/fastp#low-complexity-filter).
 
-There are two options for long-read quality filtering: [`Filtlong`](https://github.com/rrwick/Filtlong) and [`nanoq`](https://github.com/esteinig/nanoq).
+There are two options for long-read quality filtering: [`Filtlong`](https://github.com/rrwick/Filtlong) and [`nanoq`](https://github.com/esteinig/nanoq), with `nanoq` being the default option.
```

Comment:
```suggestion
There are two options for long-read quality filtering: [`Filtlong`](https://github.com/rrwick/Filtlong) and [`Nanoq`](https://github.com/esteinig/nanoq), with `Nanoq` being the default option.
```

Reply:
I prefer `nanoq`. That's the default name from the https://github.com/esteinig/nanoq